test/01_DopeWars_exerciser_test.py

```diff
@@ -41,10 +46,83 @@ async def populated_registry(ctx_factory):
         calldata=[len(signers), sample_data])
     return registry
 
-@pytest.mark.asyncio
+
+@pytest.fixture(scope='module')
+# @pytest.mark.parametrize('account_factory', [dict(num_signers=NUM_SIGNING_ACCOUNTS)], indirect=True)
+async def game_factory():
```

Comment:
can this test be updated to use the cached context similar to `01_DopeWars_contract_test`?